common/llvm.saw

```diff
@@ -49,6 +65,8 @@ let variable (type : LLVMType) (name : String) (v : { fresh : Term, t : Term}) =
 // SetupValue and Term for the variable.
 let alloc (rw : Term) (type : LLVMType) (name : String) (v : { fresh : Term, t : Term}) = do {
   v' <- variable type name v;
+  //This pointer cannot is not uninitialized.
+  err <- if (eval_bool {{ rw == 2 }}) then error else noerror;
```

Comment:
Here we make sure that crucible_fresh_pointer is not called by the alloc function.